### ISSUE TITLE
[Designer-> Header Controls-> Help]: 'Close' button is not visible on the 'Help' dialogue until user hover on it with 'mouse'.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7638]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS Version: 21H2 (OS build 22000.318)
Browser Version: 105.0.1300.0 (Official build) dev(64-bit)
Screen Reader: Narrator
URL: [Adaptive Cards] (https://adaptivecards.io/)

Repro Steps:
1. Launch the URL: [Adaptive Cards] (https://adaptivecards.io/)
2. Navigate to header control ‘Designer’ using tab key and activate it through enter key.
3. Navigate to the Help button using the TAB key.
4. Observe the issue for ‘Close’ button present on the Help dialog.

Actual Result:
Users are able see only the tool tip of the ‘close’ button, when focus moves there. and they cannot see the ‘accessible’ name of the control on the dialogue.

Expected Result:
Users should be able to see the ‘Close’ Button present on the dialogue box.

Observation:
Same issue is present for 'New Card' Button present on the Main Landmark.
PFA: Content present under New Card button is not visible.png

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.04.13
 
User Experience:
Cognitive users will get confused to access the control, if they cannot see the control on the page properly.

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)

### Screenshots

https://user-images.githubusercontent.com/104482612/177114413-12574a76-b409-478a-b75e-1bed3f6b2164.mp4
<img width="960" alt="Content present under New Card is not visible" src="https://user-images.githubusercontent.com/104482612/177114472-518205d8-52af-492d-a308-f7010c2567c8.png">

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
This issue is not repro on the environment below hence closing this issue.

Test Environment:

URL: https://adaptivecards.io
OS Version: 23H2(OS Build 25931.1000)
Browser Version: Edge Dev (118.0.2060.1)

Snippet:

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/2cbc29f5-907a-4bfe-b5f6-7bf61ee2b5dd)
